### PR TITLE
#191 add versions for NetPyNE, Neuron, NetPyNE-UI

### PR DESCRIPTION
**Pull request by Muhaddatha**

### CODE REVIEW

File: webapp/components/general/Dialog.js [hunk 1]
```diff
@@ -58,7 +64,8 @@ const AboutContent = withStyles(styles)(({ classes }) => (
 
     <Box m={1}>
       <Typography variant="body2" color={secondaryColor}>
-        NetPyNE-UI is being developed in collaboration with:
+        NetPyNE-UI is being developed by the State Univeristy of New York Downstate (<Link href="http://dura-bernal.org/" target="_blank">Dura-Bernal Lab</Link>)
```
Comment (lrebscher) [inline webapp/components/general/Dialog.js:67]: "University"

File: webapp/components/general/Dialog.js [hunk 2]
```diff
@@ -25,9 +25,15 @@ const AboutContent = withStyles(styles)(({ classes }) => (
   <Paper className={classes.paper}>
     <img width="250" src={logoNetpyne} />
     <Box m={1}>
-      <Typography variant="h5" style={{ color: secondaryColor }}>
+      <Link variant="h5" style={{ display: 'block'}} href="https://github.com/MetaCell/NetPyNE-UI" target="_blank">
         NetPyNE-UI v0.7.0
-      </Typography>
+      </Link>
+      <Link variant="h5" style={{ display: 'block'}} href="https://github.com/Neurosim-lab/netpyne" target="_blank">
+        NetPyNE v0.9.6
```
Comment (lrebscher) [inline webapp/components/general/Dialog.js:32]: v1.0.0.2 is the current version

**Review by lrebscher — CHANGES_REQUESTED** [review]
looks good 👍 
just some minor notes

File: webapp/components/general/Dialog.js (hunk 2 above)
Comment (pgleeson) [reply webapp/components/general/Dialog.js:32]: These will rapidly get out of date if they're not automatically generated. Maybe pull from https://github.com/MetaCell/NetPyNE-UI/blob/master/requirements.txt?

Comment (lrebscher) [reply webapp/components/general/Dialog.js]: I see what you mean and I created a [separate issue](https://github.com/MetaCell/NetPyNE-UI/issues/378) for this

**Review by lrebscher — APPROVED** [review]

**Pull request merged**